### PR TITLE
DDP-6504 | sample information tab

### PR DESCRIPTION
**Pull request by gmakharat**
Ticket: https://broadinstitute.atlassian.net/browse/DDP-6504

**Things have done:**

- Separated liquibase directory for rgp.
- Added liquibase file to add `hide_samples_tab` boolean column, set `hide_samples_tab` to true for `RGP`.
- Created DAO/DTO class for InstanceSettings.
- Refactored(not fully) existing `InstanceSetting` class.

### CODE REVIEW

File: src/main/resources/liquibase/rgp/DDP-6504_hide-sample-information-tab.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SimoneMaiwald) [inline src/main/resources/liquibase/rgp/DDP-6504_hide-sample-information-tab.xml:1]: nice! I like the new study folder 👍

**Review by SimoneMaiwald — APPROVED** [review]

File: src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java [hunk 2]
```diff
@@ -84,7 +85,10 @@ public void accept(PubsubMessage pubsubMessage, Context context) throws Exceptio
                     int lastKitId = 0;
                     UPSKit kit = null;
                     logger.info("tracking ups ids for " + ddpInstance.getName());
-                    boolean gbfShippedTriggerDSSDelivered = InstanceSettings.getInstanceSettings(ddpInstance.getName(), conn).isGbfShippedTriggerDSSDelivered();
+                    InstanceSettingsDto instanceSettings = new InstanceSettings().getInstanceSettings(ddpInstance.getName());
```
Comment (pegahtah) [inline src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java]: I think that won't work for CF. you need to pass the same connection so it knows how to connect to DB and get the instance settings

File: src/main/java/org/broadinstitute/dsm/db/InstanceSettings.java [hunk 3]
```diff
@@ -68,59 +77,22 @@ public InstanceSettings(List<Value> mrCoverPdf, List<Value> kitBehaviorChange, L
...
-        });
+    public InstanceSettings() {
 
-        if (results.resultException != null) {
-            throw new RuntimeException("Error getting list of realms ", results.resultException);
-        }
-        return (InstanceSettings) results.resultValue;
     }
 
-    public static InstanceSettings getInstanceSettings(@NonNull String realm, Connection conn) {
```
Comment (pegahtah) [inline src/main/java/org/broadinstitute/dsm/db/InstanceSettings.java:101]: This is what was used before @gmakharat for reference

File: src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java (hunk 2 above)
Comment (gmakharat) [reply src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java:88]: @pegahtah does it use different database or?

**Review by pegahtah — CHANGES_REQUESTED** [review]
I am going to stop this from being merged by submitting the `request changes` feedback. So it won't get merged accidentally and mess with CF

**Review by pegahtah — APPROVED** [review]

**Pull request merged**